Content.Server/Kitchen/EntitySystems/MicrowaveSystem.cs

```diff
@@ -108,6 +108,9 @@ private void OnCookStop(Entity<ActiveMicrowaveComponent> ent, ref ComponentShutd
             if (!TryComp<MicrowaveComponent>(ent, out var microwaveComponent))
                 return;
 
+            foreach (var item in _container.GetContainer(ent, "microwave_entity_container").ContainedEntities)
```

Comment:
Is `_container.GetContainer(ent, "microwave_entity_container")` identical to `microwaveComponent.Storage`?

Reply:
> Is `_container.GetContainer(ent, "microwave_entity_container")` identical to `microwaveComponent.Storage`?

Yes, my bad. I redid it